src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -36,14 +36,13 @@ case $OSName in
         ;;
 
     Linux)
-        source /etc/os-release
-        if [ "$ID" == "centos" ]; then
+        if [ "$(cat /etc/*-release | grep -cim1 centos)" -eq 1 ]; then
             __PUBLISH_RID=centos.7-x64
-        elif [ "$ID" == "rhel" ]; then
+        elif [ "$(cat /etc/*-release | grep -cim1 rhel)" -eq 1 ]; then
```

Comment:
We used `/etc/os-release` because that's what downstream tools like `cli` use to compute the RID.  If we start inspecting other files like `/etc/lsb-release` we can end up in a place where the thing we compute in this script isn't going to match what `cli` might compute (for example on a system that has an `lsb-release` file but on `os-release` file).  That may not be great,.